ext/util/math.c

```diff
@@ -200,11 +197,8 @@ php_driver_parse_bigint(char *in, int in_len, cass_int64_t *number TSRMLS_DC)
   }
 
   if (errno == ERANGE) {
-    if (*number == INT64_MAX) {
-      zend_throw_exception_ex(php_driver_range_exception_ce, 0 TSRMLS_CC, "Value is too big for bigint: '%s'", in);
-    } else {
-      zend_throw_exception_ex(php_driver_range_exception_ce, 0 TSRMLS_CC, "Value is too small for bigint: '%s'", in);
-    }
+    zend_throw_exception_ex(php_driver_range_exception_ce, 0 TSRMLS_CC,
+      "value must be between %lld and %lld, %s given", INT64_MIN, INT64_MAX, in);
```

Comment:
The arg name for all of the number class constructors is 'value'. So, making this lower-case and making it consistent with range error messages in the various types seemed like a good thing.